rest/doc_api.go

```diff
@@ -644,6 +644,10 @@ func (h *handler) handleGetLocalDoc() error {
 	}
 	value[db.BodyId] = "_local/" + docid
 	h.writeJSON(value)
+	base.Audit(h.ctx(), base.AuditIDDocumentRead, base.AuditFields{
+		base.AuditFieldDocID:      docid,
+		base.AuditFieldDocVersion: value[db.BodyRev], // this value might not be populated if this is a doc that starts with _sync
+	})
```

Comment:
The most common use case for _local documents is CBL checkpointing.  I feel like that shouldn't get bundled in with DocumentRead.  I prefer to remove, and revisit as part of a potential local doc audit event, if we think that's merited.